Dapper.Contrib/SqlMapperExtensions.cs

```diff
@@ -1001,7 +1007,7 @@ public int Insert(IDbConnection connection, IDbTransaction transaction, int? com
                 if (!first)
                     sb.Append(", ");
                 first = false;
-                sb.Append(property.Name);
+                sb.Append($"\"{property.Name}\"");

```

Comment:
isn't this the same as `adapter.AppendColumnName(sb, property.Name)`?

Reply:
Yep, not sure how I missed this one!